tensorbay/client/dataset.py

```diff
@@ -77,10 +77,8 @@ def __init__(self, name: str, dataset_id: str, gas_client: "GAS", *, status: Sta
         self._client = gas_client._client  # pylint: disable=protected-access
         self._status = status
 
-    def _commit(self, message: str, tag: Optional[str] = None) -> str:
-        post_data: Dict[str, Any] = {
-            "message": message,
-        }
+    def _commit(self, message: str, description: str, tag: Optional[str] = None) -> str:
+        post_data: Dict[str, Any] = {"message": message, "description": description}
```

Comment:
we still need to check `description` like `if description:`, if it is `""`, do not put it into `post_data` to reduce the package size.